sponsor.html

```diff
@@ -0,0 +1,316 @@
+        app_id: "i0aisamb"
+      });
+    </script>
+
+    <script>
+      (function(h,o,t,j,a,r){
+          h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+          h._hjSettings={hjid:1902775,hjsv:6};
+          a=o.getElementsByTagName('head')[0];
+          r=o.createElement('script');r.async=1;
+          r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+          a.appendChild(r);
+      })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+    </script>
+</head>
+<body>
+
+
+
+  <!-- navbar -->
+  <nav id="navbar" class="navbar navbar-expand-lg navbar-light sticky-top">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav text-center mx-auto">
+        <li class="nav-item">
+          <a class="nav-link" href="#about">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#registration">Registration</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#faq">FAQ</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="sponsor.html">Sponsor</a>
+        </li>
+      </ul>
+    </div>
+   </nav>
+
+  <div class="container-fluid">
+    <article>
+      <!-- Demographics Section -->
+      <div class="demographics">
+        <div class="row">
+          <div class="col-lg-4 col-sm-6 col-10 mx-auto p-5">
+            <canvas id="edulevels" aria-label="Educational Level Breakdown Chart" role="img"></canvas>
+          </div>
+          <div class="col-lg-8">
+            <div class="heading-lavender pl-3 d-lg-block d-none">
+              <h2>Demographics</h2>
+            </div>
+            <div class="row mb-lg">
+              <div class="col-lg-5 p-0">
+                <div class="demo-legend demo-section stretch edu m-lg-2">
+                  <span class="demo-name text-left edu-text"><span class="edu-dot"
+                      style="background: #00FFFF"></span>Undergraduate</span>
+                  <span class="demo-stat">61%</span>
+                  <br>
+                  <span class="demo-name text-left"><span class="edu-dot" style="background: #FF5789"></span>High
+                    School</span>
+                  <span class="demo-stat">23%</span>
+                  <br>
+                  <span class="demo-name text-left"><span class="edu-dot" style="background: #2D2D2D"></span>Graduate
+                    School</span>
+                  <span class="demo-stat">0.6%</span>
+                  <br>
+                  <span class="demo-name text-left"><span class="edu-dot" style="background: #A88AA8"></span>Middle
+                    School</span>
+                  <span class="demo-stat">1.7%</span>
```

Comment:
on mobile, there's a lot of space to the left and right & they aren't centered within the div